examples/workflows/plot_simulate_evoked.py

```diff
@@ -151,6 +151,14 @@
 dpls_sync[trial_idx].copy().smooth(window_len).scale(scaling_factor).plot()
 net_sync.cell_response.plot_spikes_hist()
 
+###############################################################################
+#.. warning::
```

Comment:
see how it is rendered [here](https://2153-168215891-gh.circle-artifacts.com/0/html/auto_examples/workflows/plot_simulate_evoked.html#sphx-glr-auto-examples-workflows-plot-simulate-evoked-py). You actually want it to look like this:

![image](https://user-images.githubusercontent.com/15852194/155816863-6cf3d1ef-e2d2-454d-87f6-6a0da8160ec7.png)

also, as a matter of style, I would recommend breaking the line before 80 characters (pep8). For some reason our linters don't run on the examples ... which is a little unfortunate. The following worked for me:

```sh
# .. warning::
#    - Always look at dipoles in conjunction with raster plots and spike
#      histogram to avoid misinterpretation.                
#    - Run multiple trials of your simulation to get an average of different
#      drives seeds before drawing conclusions.
```

Reply:
@jasmainak It looks to me that it rendered the same in both examples but I have adjusted the code to break by 80 characters, shall I push it again to you?

Reply:
you're missing the orange background:

![image](https://user-images.githubusercontent.com/15852194/155856827-488e4c7b-28a3-4042-a5b1-e606e8cb18c0.png)

Yes, push an update and let us know!

Reply:
![Capture](https://user-images.githubusercontent.com/14954705/155857336-db637419-fe33-41b7-be0a-cb6d38ea08ca.PNG)
@jasmainak weird, I see it with the orange background on my end

Reply:
I actually see the orange on my end as well, maybe it's browser specific?

Reply:
my bad, it's the mu-matrix plugin on my browser. It got over zealous :)

Reply:
@jasmainak would this pull request work? or shall I make a new one?

Reply:
No need to open a new one! If there are changes you want to undo, it is typically better to make a commit that returns the code to the previous state of the repository. That way all of your work for a specific PR (even if it is undone) gets tracked in the same place.